transifex/push.py

```diff
@@ -25,18 +25,28 @@
 MESSAGE = 'Update translation strings'
 
 
-def push(clone_url, repo_owner, merge_method=DEFAULT_MERGE_METHOD):
+def push(clone_url, repo_owner, merge_method=DEFAULT_MERGE_METHOD, no_commit=False):
     """Extracts translations for the given repo, commits them, pushes them to GitHub, opens a PR, waits for status
         checks to pass, merges the PR, deletes the branch, and pushes the updated translation files to Transifex.
     """
     with repo_context(clone_url, repo_owner, BRANCH_NAME, MESSAGE, merge_method=merge_method) as repo:
         logger.info('Extracting translations for [%s].', repo.name)
         repo.extract_translations()
-        repo.commit_push_and_open_pr()
 
-        if repo.pr and repo.merge_pr() and repo.pr.is_merged():
-            logger.info('Pushing translations to Transifex for [%s].', repo.name)
-            repo.push_translations()
+        if not no_commit:
```

Comment:
Nit: I'd prefer to call this something positive, like skip_commit, as that's easier to read in if statements.

Reply:
Actually, no_commit is fine, but it'd be easier to read if we switched the if statetments around.
```
if no_commit:
  push_translations_to_transifex(repo)
else:
  # The other case...
```

Reply:
have changed to skip_commit.